dist/iidx/iidxhook-09.conf

```diff
@@ -10,7 +10,7 @@ eamuse.pcbid=0101020304050607086F
 # EAMID
 eamuse.eamid=0101020304050607086F
 
-# Fix stretched BG videos on newer GPUs. Might appear on Red and newer
+# Fix stretched BG videos on newer GPUs. Might appear on SIRIUS and older. On 9th and 10th style this issue may only affect older BGAs (from 1st-3rd style)
```

Comment:
```suggestion
# Fix stretched BG videos on newer GPUs. Might appear on 9 to 17
```

I think we don't have to be that specific in the config file here. Maybe worth mentioning that last part about some BGAs only affected on 9 to 10 in a comment in the code.

Reply:
I figured some users might not realize their videos are stretched since it doesn't affect all of them, this would spur them to check an affected song to see if they need to enable the option. (As opposed to later styles where all videos are affected) But that can be removed if you think it's unnecessary

Reply:
Ok, fair point that it might not be obvious to the user. Let's keep your version then.

Can you also adjust that description in `src/main/iidxhook-util/config-gfx.c`?

Reply:
No problem